src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx

```diff
@@ -69,6 +69,8 @@ const ActionsListItem = ({
   //   motionState,
   // );
 
+  const status = ListItemStatus.Defused;
```

Comment:
Is this a temporary solution? I'm not certain on how "NeedAction/NeedAttention" were used in the dapp but if we don't plan on using them they could be removed from the enum.

Reply:
Yes, this is temporary (we don't have `status` field on `ColonyAction` yet)